source/manual/howto-remove-change-note-from-whitehall.html.md

```diff
@@ -0,0 +1,39 @@
+---
+owner_slack: "#2ndline"
+title: Remove a change note in Whitehall
+section: Whitehall
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-02-09
+review_in: 6 months
+---
+
+Change notes are called `editorial_remarks` in Whitehall. An Edition can 
+have multiple editorial remarks, but in the Publishing API an Edition can 
+only have one single change note. Not all editorial remarks are propagated 
+to the Publishing API, only ones marking major changes. The Publishing API 
+then creates a list of all the change notes from all versions of the edition
+and presents them to the Content Store. You can read more about this in the 
+Publishing API [docs](https://docs.publishing.service.gov.uk/apis/publishing-api/model.html#changenote).
+
+### Remove a Change Note
+
+Steps to undertake when someone has requested that you remove a change note 
+from Whitehall: 
+
+1. Obtain the ID of the edition on which the change note was created. 
+2. Create a migration in Whitehall where you search for the edition by ID 
```

Comment:
it is best practice to number these all `1.` so that adding/removing a step is easier to do in the future.

Markdown will automatically generate the correct numbering sequence.